### DISCUSSION TITLE
Class "Livewire\InvalidArgument" not found

### DISCUSSION THREAD

**Discussion by firebed** [livewire/livewire#3864] [7. Bugs (archived)] [answered]
After the last update v2.6.6 I am getting this error. Maybe it was meant to be InvalidArgumentException I don't know.

**Comment by joshhanley** [comment, marked as answer]
@firebed please share a screenshot of the error and whether you performed any actions to trigger it (like clicking a button or typing in an input, etc).

Thanks!

**Reply by firebed** [reply]
@joshhanley There was no user input or any kind of action. After the update I just refreshed the page.
![image](https://user-images.githubusercontent.com/37473473/134787196-eb99ec3e-5d17-4d49-a451-473f31849fc3.png)

**Reply by joshhanley** [reply]
@firebed thanks for the screenshot, that helps! Ah yep I think I know what's going on.

**Reply by firebed** [reply]
@joshhanley Is there anything I should do to fix the issue?

**Reply by joshhanley** [reply]
@firebed nah it's a bug, I'm currently working on fixing it 🙂

**Comment by joshhanley** [comment]
@firebed this has been fixed and tagged in v2.6.7 🙂

**Reply by firebed** [reply]
@joshhanley Thanks!
